README.md

```diff
@@ -1,2 +1,7 @@
 # identity-serializer-fst
-We have a requirement to have a fast serializer on behalf of Java serializer for performance improvements
+Fast serialization implementation that can be used on behalf of default java serializer
```

Comment:
"Fast serialization implementation that can be used instead of default java serializer, which gives better DB performance"